src/System.Net.HttpListener/src/System/Net/HttpListenerResponse.cs

```diff
@@ -27,20 +25,12 @@ public WebHeaderCollection Headers
             }
         }
 
+        public Encoding ContentEncoding { get; set; }
```

Comment:
Ah, I see, that default was only being provided in the managed implementation, not the Windows, which is your point.  Ok.